db/schema.rb

```diff
@@ -44,38 +44,43 @@ def up
 
     end
 
-    ## Github tables:
-    create_table? :pullrequests do
-      Integer :pr_number, null: false
+    ## Github/Gitlab tables:
+    create_table? :changerequests do
+      Integer :change_number, null: false
       String :title, null: false
       String :state, null: false
       String :url, null: false
       String :rname, null: false
       String :oname
       DateTime :created_at
       DateTime :updated_at
+      String   :origin
       DateTime :closed_at
       DateTime :merged_at
 
-      primary_key [:rname, :pr_number], name: :pr_identifier
+      primary_key [:rname, :change_number], name: :change_identifier
     end
 
     # stores trend of a specific repo:
-    create_table? :pulltrends do
+    create_table? :changetrends do
       DateTime :time
       Integer :open
       Integer :closed
       String :rname, null: false
       String :oname
+      String :origin
 
-      primary_key [:time, :rname], name: :pulltrend_identifier
+      primary_key [:time, :rname, :origin], name: :changetrend_identifier
```

Comment:
This is still missing `:oname` in the identifier to be truly unique or? Not that I blame my own mistakes on you :grimacing:

Reply:
Done